request/load_balancer_reverse_proxy.rst

```diff
@@ -25,39 +25,19 @@ and which reverse proxy IP addresses will be doing this type of thing:
 
 .. configuration-block::
 
-    .. code-block:: yaml
-
-        # app/config/config.yml
-        # ...
-        framework:
-            trusted_proxies:  [192.0.0.1, 10.0.0.0/8]
-
-    .. code-block:: xml
-
-        <!-- app/config/config.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:framework="http://symfony.com/schema/dic/symfony"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
-                                http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+   .. code-block:: diff
```

Comment:
Currently, `diff` does not work (we need #7806 for it). We should rewrite the example a bit and use the common `php` code block instead.